### ISSUE TITLE
Psalm 118 Hebrew acrostic stanza headings?

### ISSUE THREAD

**Issue by DavidHaslam** [MarjorieBurghart/VulgateGlaire#10]
Currently the **acrostic headings** were inserted into the verse `1 MOD 8` text for each of the 22 stanzas.

Are they really proper headings in the **printed** edition?

_If so, I could easily adjust the USFM tags to implement this in the SWORD module._

![screenshot 2017-10-09 13 32 47](https://user-images.githubusercontent.com/16325414/31338276-6b87c524-acf6-11e7-8df1-e515211c72ec.png)

**Comment by DavidHaslam** [comment]
This is what Psalm 118 looks like in the [Bible Vigouroux](http://jesusmarie.free.fr/bible_vigouroux_psaumes.html) online.

![screenshot 2017-10-09 13 44 00](https://user-images.githubusercontent.com/16325414/31338712-4065a97c-acf8-11e7-80e0-5710cf4506f0.png)

It's evident that the **verse number** tag is actually _after_ the acrostic letter name.
In SWORD, the only way this can be achieved would be by making each acrostic a **heading**, thus:
```
\c 118
\qa ALEPH :
\p
\v 1 \it Alleluia.\it* (Bien)Heureux ceux qui sont immaculés dans la voie, qui marchent dans la loi du (le) Seigneur.
\v 2 (Bien)Heureux ceux qui étudient ses ordonnances (témoignages), et qui le cherchent de tout leur cœur.
\v 3 Car ceux qui commettent l’iniquité ne marchent pas (n’ont pas marché) dans ses voies.
\v 4 Vous avez ordonné que vos commandements soient très exactement gardés.
\v 5 Puissent mes voies être dirigées de telle sorte, que je garde vos ordonnances (pour garder vos justifications) !
\v 6 (Alors) Je ne serai point confondu, lorsque j’aurai sous les yeux tous vos préceptes (commandements).
\v 7 Je vous louerai dans la droiture de mon cœur, de ce que j’ai appris les préceptes (jugements) de votre justice.
\v 8 Je garderai vos ordonnances ; (justifications :) ne m’abandonnez pas entièrement.
\qa BETH :
\p
\v 9 Comment le (un) jeune homme corrigera-t-il sa voie ? En accomplissant (gardant) vos paroles.
...
```

Refer to the [USFM documentation](http://paratext.org/usfm) for **poetry**.
NB. Even though Psalms (etc) in Glaire is **not** formatted as poetry, we can still use the `\qa ` marker.

Once these 22 acrostics are made into headings, there is no need to retain the ` :" that served to separate them from the verse text.

**Comment by MarjorieBurghart** [comment]
Yes, there's an issue with the ODD here, it doesn't handle such phenomena I'm afraid. Since it is a shared ODD between this French Vulgate and a KJV Bible, I'll have to discuss this with the creator of the ODD. 
We try to use the same schema for those Bibles, to make life easier on people who might want to convert them in the future.

**Comment by DavidHaslam** [comment]
There's a long history of Bible software and XML schema that in the earliest days even tied developers down to a fixed versification that reflected the KJV. We've come a long way since then.

The **OSIS XML** schema was developed by the **Bible Technologies Group**, though their website seems to have gone AWOL again recently. (This has happened twice before).

OSIS = **Open Scriptures Information Standard**.

The latest released version of the OSIS DTD is **2.1.1**
We have copies available in CrossWire and there's also one at eBible.org
Our own custom enhanced schema is documented in our developers' wiki.
https://crosswire.org/wiki/OSIS_211_CR

FYI. Many Bible translators use **ParaTExt** software which still supports import and export of USFM.
From **ParaTExt 7.0** the internal Bible data is stored in an XML format called **USX**.
From **ParaTExt 8.0** the software becomes available with an open source license.

**Comment by DavidHaslam** [comment]
The acrostics in **Lamentations** I propose to tag at character level using `\qac ...\qac*`.
I replaced the USFM italics _character style_ markup by this _semantic_ markup.

We'll then see how these fare after converting to OSIS and rebuilding the module.
Whether they display as italics would depend on the app and that some apps also support style sheets.

**Comment by DavidHaslam** [comment]
In the OSIS XML, the acrostics in Lamentations then become like `<hi type="acrostic">Aleph.</hi>`.

Currently, SWORD does not assign any _different_ character style for this **defined** type attribute value.
It displays as normal text.

Front-end apps such as **Xiphos** do support CSS, so for that at least, it's feasible to allocate a style for the text wrapped in this hi type.

**Comment by DavidHaslam** [comment]
**Psalm 118** acrostics now display as stanza headings in my latest module build.

![screenshot 2017-10-10 15 31 03](https://user-images.githubusercontent.com/16325414/31392089-417a4c22-add0-11e7-8cca-1c38d92dd307.png)

btw. The colon separators became _redundant_ and have been removed.

**Comment by DavidHaslam** [comment]
Incidentally, did the printed edition also include the 22 actual Hebrew letters as well?
# אבגדהוזחטיכלמןנסעפצקרשת
